src/Controls/src/Core/Device.cs

```diff
@@ -165,7 +169,7 @@ public static void StartTimer(TimeSpan interval, Func<bool> callback)
 		[EditorBrowsable(EditorBrowsableState.Never)]
 		public static Assembly[] GetAssemblies()
 		{
-			return PlatformServices.GetAssemblies();
+			return AppDomain.CurrentDomain.GetAssemblies();
```

Comment:
All of the platforms just call this exact method `AppDomain.CurrentDomain.GetAssemblies()` so it didn't seem necessary here to have this on the platform services interface